src/ralph/backends/plugins/fs/source/fs.py

```diff
@@ -10,7 +10,7 @@
 from typing import IO, Iterable, Iterator, Union
 from uuid import uuid4
 
-from ralph.backends.data.base import (
+from ralph.factories.base import (
```

Comment:
No need to create a directory `factories`, I think it can stay inside `backends.data`